### PR TITLE
Add get interface call on switches

### PR DESCRIPTION
**Pull request by fbouliane**
* Compliance test
* added to the switch API
* All declination of switches now support this

### CODE REVIEW

File: tests/adapters/switches/remote_test.py [hunk 1]
```diff
@@ -548,6 +548,42 @@ def test_get_vlans(self):
...
+        interface = self.switch.get_interface('ethernet 1/4')
+
+        assert_that(interface.name, equal_to("ethernet 1/4"))
+        assert_that(interface.shutdown, equal_to(False))
+        assert_that(interface.port_mode, equal_to(TRUNK))
+        assert_that(interface.access_vlan, equal_to(None))
+        assert_that(interface.trunk_native_vlan, equal_to(2999))
+        assert_that(interface.trunk_vlans, equal_to([3000, 3001, 3002]))
+
+    def test_get_nonexistent_interface(self):
```
Comment (idjaw) [inline tests/adapters/switches/remote_test.py]: minor: Would be good to specify what the expectation is for this test. "test_get_nonexistent_interface_raises"

Comment (idjaw) [inline tests/adapters/switches/remote_test.py]: other than that :+1:

**Comment by idjaw** [comment]
:+1:

**Comment by godp1301** [comment]
:+1:

**Pull request merged**